vars/main.yml

```diff
@@ -65,6 +65,10 @@ patroni_etcd_hosts: []  # list of servers of an existing etcd cluster
 #  - { host: "10.128.64.140", port: "2379" }
 #  - { host: "10.128.64.142", port: "2379" }
 #  - { host: "10.128.64.143", port: "2379" }
+patroni_etcd_namespace: "/service/"  # (optional) etcd namespace for patroni cluster
+patroni_etcd_username:  # (optional) username for etcd authentication
+patroni_etcd_password:  # (optional) password for etcd authentication
+patroni_etcd_protocol:  # (optional) http or https, if not specified http is used
```

Comment:
```suggestion
patroni_etcd_namespace: "/service/"  # (optional) etcd namespace for patroni cluster
patroni_etcd_username:  "" # (optional) username for etcd authentication
patroni_etcd_password:  "" # (optional) password for etcd authentication
patroni_etcd_protocol:  "http"
```

Reply:
> patroni_etcd_protocol

Please pay attention to the confd parameters:

https://github.com/vitabaks/postgresql_cluster/blob/master/roles/confd/templates/confd.toml.j2#L12

Reply:
> Please pay attention to the confd and vip-manager parameters

Sure, I'll take a look.